Makefile

```diff
@@ -10,7 +11,11 @@ vpath % src
 
 # .PHONY all
 
-all: calculator
+all: main

```

Comment:
issue with dependency is my best guess, change back to calculator or rename calculator to main